_includes/post-preview.html

```diff
@@ -0,0 +1,31 @@
+{% assign post = include.post %}
+
+{%- capture excerpt_link_url -%}
+  {%- if post.excerpt_link_url -%}
+    {{ post.excerpt_link_url }}
+  {%- else -%}
+    {{ site.baseurl }}{{ post.url }}
+  {%- endif -%}
+{%- endcapture -%}
+
+<article id="{{ post.id | default: post.title | slugify }}" class="post-preview">
+  <header class="post-preview__header">
+    <p class="post-category site-subheading">
+      {{ post.tags[0] }}
+    </p>
+
+    <{{ include.heading | default:"h2" }} class="post-preview__title">
```

Comment:
**question**: Is this heading being overridden elsewhere?

Reply:
It is. The post preview headers need to be h3's on the [what's new page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-whats-new-post-previews/about/whats-new/) and h2's on the [all news and updates page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-whats-new-post-previews/about/whats-new/all/).